__tests__/utils.tsx

```diff
@@ -404,23 +419,30 @@ test('NotFoundResponse', async () => {
   await expectIsNotFoundResponse(createNotFoundResponse())
 })
 
+function serverMock(url_e: string, body_e: string) {
+  server.use(
+    rest.get(url_e, (_req, res, ctx) => {
+      return res.once(ctx.status(200), ctx.body(body_e))
+    })
+  )
+}
```

Comment:
I would move this function to the file `__tests__/_helper.ts` from which you can include it. It would only work after you have moved `serverSetup()` to `jest.setup.ts` (see comment above).